api/src/services/authorization.ts

```diff
@@ -396,7 +397,9 @@ export class AuthorizationService {
 					for (const field of requiredPermissions[collection]) {
 						if (field.startsWith('$FOLLOW')) continue;
 						const fieldName = stripFunction(field);
-						if (!allowedFields.includes(fieldName)) {
+						const originalFieldName = collection === rootCollection ? aliasMap?.[fieldName] ?? fieldName : fieldName;
```

Comment:
Lets rework this a hair to be a little more legible

```suggestion
let originalFieldName = fieldName;

if (collection === rootCollection && aliasMap?.[fieldName]) {
  originalFieldName = aliasMap[fieldName];
}
```

Reply:
^ pseudo code

Reply:
It's pseudo code that works perfectly! 👍